Sources/AppStoreConnectCLI/Helpers/File+Helper.swift

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+import Files
+
+extension File {
+    static func folderAndFilename(from path: String) -> (String, String) {
```

Comment:
I don't think we actually need this method. It is also an extension on `File` which isn't accurate.